flambe/utils/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ['config']
```

Comment:
Could you add a direct import:

```
from flambe.utils.config import generate_config_from_template

__all__ = ['generate_config_from_template']
```

That way people can import from flambe.utils directly :)